packages/solidity-contracts/test/behaviors/erc20GatewayV4.behavior.test.ts

```diff
@@ -1340,6 +1353,10 @@ export function behavesLikeErc20GatewayV4(fixture: () => Promise<Env>) {
                 RATE_LIMIT_DURATION
               );
 
+            await erc20Gateway
+              .connect(deployer)
+              .updateRateLimitStatus(token, true);
+
```

Comment:
good to add a couple of tests to make sure the state is updated and the status update event is emitted.